classes/entity/enemy/cheep_cheep/cheep_cheep.gd

```diff
@@ -88,6 +88,7 @@ func _on_WaterCheck_area_exited(_area):
 
 
 func _hurt_struck(body):
+	._hurt_struck(body)
```

Comment:
What problem does this resolve?

Reply:
...Oh, hm. That was meant to just call the struck sound, but it looks like there's a whole bunch of extra side effects in the base function....

Ah, the joys of inheritance 😒